.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+repos:
+  - repo: local
+    hooks:
+      - id: flake8
+        name: Lint Python with Flake8
+        entry: make lint
+        language: system
+        types: [python]
+      - id: circleci-config
+        name: Validate CircleCI Config
+        entry: bash circleci-validate.sh
```

Comment:
You'll need the `circleci-validate.sh` script in your repo as well, otherwise this will fail